requirements-train.in

```diff
@@ -0,0 +1 @@
+tensorflow==1.10.0
```

Comment:
If you're not using this file, delete it.

Reply:
Ok, sure